sale_payment_sheet_financial_risk/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. Go to *Customers > Financial Risk*
+#. Choose options to compute in credit limit.
```

Comment:
Which options? Please complete this

Reply:
Go to *Customers > Financial Risk*
and you can see all options to set...

Reply:
Yes, but write a bit more about it. Specifically, about the check that enables to check this amount, and the other limit. Also a bit about the sign.

Reply:
Changes done